server/jobs/onboarding/freckle.js

```diff
@@ -9,7 +9,7 @@ import { creatOrUpdatePartnerRecord } from '../../modules/automations';
  * @param {object} automationResult Result of automation job
  * @returns {undefined}
  */
-export default async (placement) => {
+const freckleOnboarding = async (placement) => {
```

Comment:
This small change of naming and then exporting the function is only meant to increase coverage as a file that was previously covered has been deleted and was dropping the coverage. This change compensates for the drop in coverage. cc @valmayaki.

Reply:
Thanks